Post-PR review. (#32562)
dotnet/AspNetCore.Docs: aspnetcore/fundamentals/minimal-apis/aspnetcore-openapi.md

* Document transformers have access to the entire OpenAPI document and can be used to make global modifications to the document.
* Operation transformers apply to each individual operation (combination of path + HTTP method) and can be used to modify parameters or responses on endpoints.

> Transformers can be registered onto the document via the `UseTransformers` call on the `OpenApiOptions` object. The following snippet shows different ways to register transformers onto the document:

Reviewer: ```suggestion
Transformers can be registered onto the document via the `UseTransformer` call on the `OpenApiOptions` object. The following snippet shows different ways to register transformers onto the document:
```

});
```

### Using document transformers

Document transformers have access to a context object that includes:

* The name of the document being modified.
* The list of ApiDescriptionGroups associated with that document.
* The IServiceProvider used in document generation.

Document transformers also have mutate access to the OpenAPI document that has been generated. The following example demonstrates a document transformer that adds some information about the API to the OpenAPI document.

[!code-csharp[](~/fundamentals/minimal-apis/9.0-samples/WebMinOpenApi/Program.cs?name=snippet_documenttransformer1)]

Service-activated document transformers can also be used to implement transformers that rely on instances from DI to modify the application. The sample below demonstrates a document transformer that uses the `IAuthenticationSchemeProvider` service from the authentication layer to check if any JWT bearer-related schemes are registered in the application and add them to the OpenAPI document's top level.

[!code-csharp[](~/fundamentals/minimal-apis/9.0-samples/WebMinOpenApi/Program.cs?name=snippet_documenttransformer2)]

> Document transformers are unique to the document instance they are associated with. In the example below, a transformer registers authentication-related requirements to the `internal` document but leaves the `public` document unmodified.<!-- "example below" this is the end of the section, there is no following snippet. need to find snippet or delete reference to it -->

Reviewer 2: ```suggestion
Document transformers are unique to the document instance they are associated with. In the example below, a transformer registers authentication-related requirements to the `internal` document but leaves the `public` document unmodified.

[!code-csharp[](~/fundamentals/minimal-apis/9.0-samples/WebMinOpenApi/Program.cs?name=snippet_multidoc_operationtransformer1)]
```
